pkg/controller/workload/workload_processor.go

```diff
@@ -587,10 +603,52 @@ func (p *Processor) handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDis
 	}
 
 	_ = p.handleRemovedAddresses(rsp.RemovedResources)
+	p.compareWorkloadAndServiceWithHashName()
 
 	return err
 }
 
+// When processing the workload's response for the first time,
+// fetch the data from the /mnt/workload_hash_name.yaml file
+// and compare it with the data in the cache.
+func (p *Processor) compareWorkloadAndServiceWithHashName() {
+	var (
+		bk = bpf.BackendKey{}
+		bv = bpf.BackendValue{}
+		sk = bpf.ServiceKey{}
+		sv = bpf.ServiceValue{}
+	)
+
+	if kmeshbpf.GetStartType() != kmeshbpf.Restart {
+		return
+	}
+
+	log.Infof("reload workload config from last start")
+	kmeshbpf.SetStartType(kmeshbpf.Normal)
+
+	// The record exists in the hashName file, exists in Backend or Service bpfmap,
+	// and does not exist in cache.
```

Comment:
Confusing

Reply:
I guess you mean when the str exists in hash file:
1. if not in the userspace cache, we should delete it from bpf